bindings/zeebe/command/command.go

```diff
@@ -19,7 +18,7 @@ import (
 const (
 	// operations
 	topologyOperation         bindings.OperationKind = "topology"
-	deployWorkflowOperation   bindings.OperationKind = "deploy-workflow"
+	deployProcessOperation    bindings.OperationKind = "deploy-process"
```

Comment:
Keep the old name too, so it is backwards compatible.

Reply:
Looking at our documentation, this component is not in GA. So, it can be a breaking change. This change is OK.

https://docs.dapr.io/operations/components/certification-lifecycle/#alpha
https://docs.dapr.io/operations/components/certification-lifecycle/#beta
https://docs.dapr.io/reference/components-reference/supported-bindings/#zeebe-camunda